frontend/src/metabase/query_builder/components/filters/FilterPopover.jsx

```diff
@@ -55,6 +55,20 @@ export default class FilterPopover extends Component<*, Props, State> {
         tableMetadata: PropTypes.object.isRequired
     };
 
+    componentWillMount() {
```

Comment:
I had to add `this.commitOnEnter = this.commitOnEnter.bind(this);` here to make this work